src/lib/comms/include/sol-network.h

```diff
@@ -73,6 +73,9 @@ extern "C" {
  */
 #define SOL_INET_ADDR_STRLEN 48
 
+#ifndef AF_BT_IOTIVITY
+    #define AF_BT_IOTIVITY 31
```

Comment:
we're discussing at IRC, likely will require bigger changes, but it will pay off in the future:
 - create sol-ble.h and move BLE wrappers in there;
 - split network handling from sol-coap and sol-oic. We can keep the current functions as a helpers since socket will be the common transport, but we also work based on given buffers:
   * this allows us to use sol-socket
   * add BLE
   * load/save to files, enabling more testing like fuzzy-testers (ie: AFL)
 - sol-oic and sol-coap would handle different transports such as BLE, OIC, SMS, XMPP...